consensus/src/checkpoint.rs

```diff
@@ -69,3 +69,89 @@ pub fn is_short_range(
+        let mut genesis: ProtocolState = Default::default();
+        init_checkpoints(&mut genesis).unwrap();
+        assert_eq!(
+            genesis
+                .body
+                .consensus_state
+                .staking_epoch_data
+                .start_checkpoint,
+            StateHash::default()
+        );
+        assert_eq!(
+            genesis
+                .body
+                .consensus_state
+                .staking_epoch_data
+                .lock_checkpoint,
+            StateHash::default()
+        );
+        assert_eq!(
+            genesis
+                .body
+                .consensus_state
+                .next_epoch_data
+                .start_checkpoint,
+            StateHash::default()
+        );
+        assert_eq!(
+            genesis.body.consensus_state.next_epoch_data.lock_checkpoint,
+            Base58Encodable::from_base58("3NLoKn22eMnyQ7rxh5pxB6vBA3XhSAhhrf7akdqS6HbAKD14Dh1d")
+                .unwrap()
+        );
+    }
+
+    #[test]
+    fn test_is_short_range() {
+        let mut genesis: ProtocolState = Default::default();
+        init_checkpoints(&mut genesis).unwrap();
+        let mut c0: ProtocolStateChain = ProtocolStateChain(vec![]);
+        let mut c1: ProtocolStateChain = ProtocolStateChain(vec![]);
+        let mut c3: ProtocolStateChain = ProtocolStateChain(vec![]);
+        let mut b0: ProtocolState = Default::default();
+        b0.body.consensus_state.blockchain_length = Length(0);
+        b0.body.consensus_state.curr_global_slot = GlobalSlot {
+            slot_number: GlobalSlotNumber(0),
+            slots_per_epoch: Length(7140),
+        };
+        let mut b1: ProtocolState = Default::default();
+        b1.body.consensus_state.blockchain_length = Length(1);
+        b1.body.consensus_state.curr_global_slot = GlobalSlot {
+            slot_number: GlobalSlotNumber(1),
+            slots_per_epoch: Length(7140),
+        };
+
+        c0.push(b0).unwrap();
+        c1.push(b1).unwrap();
+        assert_eq!(is_short_range(&c0, &c1).unwrap(), true);
+        assert_eq!(is_short_range(&c1, &c0).unwrap(), true);
+
+        init_checkpoints(&mut genesis).unwrap();
+        let mut b1: ProtocolState = Default::default();
+        b1.body.consensus_state.blockchain_length = Length(2);
+        b1.body.consensus_state.curr_global_slot = GlobalSlot {
+            slot_number: GlobalSlotNumber(2),
+            slots_per_epoch: Length(7140),
+        };
+        let mut b2: ProtocolState = Default::default();
+        b2.body.consensus_state.blockchain_length = Length(667);
+        b2.body.consensus_state.epoch_count = Length(11);
+        b2.body.consensus_state.curr_global_slot = GlobalSlot {
+            slot_number: GlobalSlotNumber(667),
+            slots_per_epoch: Length(7140),
+        };
+        c3.push(b2).unwrap();
+        assert_eq!(is_short_range(&c3, &c0).unwrap(), false);
+        assert_eq!(is_short_range(&c0, &c3).unwrap(), false);
+    }
+}
```

Comment:
@hanabi1224 has already moved this part to the tests folder see [link](https://github.com/ChainSafe/mina-rs/pull/90/files). Don't you need to modify this part?

Reply:
Yeah, i think during rebase it may have added changes to this file too. these changes should not be in this PR. i will revert  them